browsers/opera_android.json

```
@@ -482,7 +482,7 @@
           "release_notes": "https://blogs.opera.com/mobile/2024/06/opera-android-version-image-generation/",
           "status": "current",
           "engine": "Blink",
-          "engine_version": "124"
+          "engine_version": "125"
```

Is it 126 even? https://forums.opera.com/topic/72570/opera-for-android-83

```suggestion
          "engine_version": "126"
```

---

Yep, just confirmed that from the user agent string!